app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.framgia.soundclound.util;
+
+import android.databinding.BindingAdapter;
+import android.graphics.PorterDuff;
+import android.support.design.widget.TabLayout;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.screen.main.TabType;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.screen.main.TabType. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports